OpenRA.Mods.Common/Traits/Infantry/TakeCover.cs

```diff
@@ -22,11 +24,27 @@ public class TakeCoverInfo : TurretedInfo
 		[Desc("Prone movement speed as a percentage of the normal speed.")]
 		public readonly int SpeedModifier = 50;
 
+		[Desc("Damage types that trigger prone state. Defined on the warheads.")]
+		public readonly string[] ProneTriggers = new string[0];
```

Comment:
`ProneTriggers` &rarr; `DamageTriggers` please.  This trait is dedicated to the prone behaviour, so the prone prefix is redundant.